stages/org.osbuild.dracut

```diff
@@ -204,8 +210,11 @@ def main(tree, options):
     for kver in kernels:
         print(f"Building initramfs for {kver}", file=sys.stderr)
 
-        subprocess.run(["/usr/sbin/chroot", tree,
-                        "/usr/bin/dracut",
+        bin="/usr/bin/dracut"
+        if initoverlayfs:
+            bin="/usr/bin/initoverlayfs-install"
```

Comment:
Maybe it's obvious but it might be worthwhile to mention in the comment of the stage that this requires the initoverlayfs package to be installed in the chroot?

Reply:
Can add

Reply:
Added via:

diff --git a/stages/org.osbuild.dracut b/stages/org.osbuild.dracut
index 5858c132..ead64a83 100755
--- a/stages/org.osbuild.dracut
+++ b/stages/org.osbuild.dracut
@@ -121,7 +121,7 @@ SCHEMA = """
     "type": "boolean"
   },
   "initoverlayfs": {
-    "description": "Use initoverlayfs rather than initramfs",
+    "description": "Use initoverlayfs rather than initramfs, requires initoverlayfs rpm to be installed",
     "type": "boolean",
     "default": false
   },